### PR TITLE
More Clear Examples for Creating Gift Cards, Accounts/Points, and Promotions

### PR DESCRIPTION
**Pull request by Tim-Jordison**
Added sections for creating gift card, account or promotion programs.

Also, adds sections on issuing ValueStores from the types of programs. This was added since the general ValueStore create endpoint contains a lot of information regarding details that are hard to wade through when creating a ValueStore from a specific gift card.

### CODE REVIEW

File: apiary-sources/endpoints/programs-create-account-program.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+---
++ Request (application/json)
+    + Headers
+    
+            {{header.authorization}}
+        
+    + Attributes
+        + programId (string, required) - {{program.programId}}
+        + name (string, optional) - {{transaction.name}}
```
Comment (tjukes) [inline apiary-sources/endpoints/programs-create-account-program.md]: `program.name`?

File: apiary-sources/endpoints/programs-create-gift-card-program.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        + metadata (number, optional) - {{program.metadata}}
+
+    + Body
+
+            {
+                "programId": "gift-cards-usd",
+                "name": "Gift Cards USD",
+                "currency": "USD",
+                "access": "secureCode",
+                minInitialValue: 500,
```
Comment (pushplay) [inline apiary-sources/endpoints/programs-create-gift-card-program.md]: JSON formatting.

File: apiary-sources/endpoints/programs-create-promotion-program.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+### Create Promotion Program [POST /programs]
+
+Create a new Promotion Program.
```
Comment (pushplay) [inline apiary-sources/endpoints/programs-create-promotion-program.md]: Promotion is pretty vague so I think you need to better define what kind of promotion is this.

File: apiary-sources/api-sections/programs.md [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+Programs act as a template to set properties of ValueStores for the type of value you wish to issue. 
+A few common examples of the types of value you may want to issue and define as Programs are:
+- Gift Cards
+- Accounts or Points
+- Promotions  
+
+Below are examples of how you'd create a Program for the common types listed above.
 
 ---
+{% include 'endpoints/programs-create-gift-card-program.md' %}
```
Comment (pushplay) [inline apiary-sources/api-sections/programs.md]: I worry the separation between the canonical REST API definition and examples isn't clear.  Hard to say for sure without this being rendered.  Maybe a different document is a better place for examples?

**Pull request merged**